transform/mattermost-analytics/models/intermediate/product/features/calls/int_calls_daily_usage_per_user.sql

```diff
@@ -0,0 +1,32 @@
+{{
+    config({
+        "materialized": "table",
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+select
+    cast(timestamp as date) as activity_date
+    , server_id
+    , user_id
+    , {{ dbt_utils.generate_surrogate_key(['activity_date', 'server_id', 'user_id']) }} as daily_user_id
+    , count_if(feature_name = 'Calls' and array_size(feature_skus) > 0) as count_calls
+    , count_calls as count_known_features
```

Comment:
A q would be - at which point the reading & interpretation of the skus (paid vs. free) is done, here or in some other place?